content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -0,0 +1,263 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Ephemeral Volumes
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+This document describes _ephemeral volumes_ in Kubernetes. Familiarity
+with [volumes](/docs/concepts/storage/volumes/) is suggested, in
+particular `PersistentVolumeClaim` and `PersistentVolume`.
+
+<!-- body -->
+
+Some application need additional storage but don't care whether that
+data is stored persistently across restarts. For example, caching
+services are often limited by memory size and can move infrequently
+used data into storage that is slower than memory with little impact
+on overall performance.
+
+Other applications expect some read-only input data to be present in
+files, like configuration data or secret keys.
+
+_Ephemeral volumes_ are designed for these use cases. Because volumes
+are created anew for each pod, pods can be stopped and restarted
+without being limited to where some persistent volume is available.
+
+Ephemeral volumes are specified _inline_ in the pod spec, which
+simplifies application deployment and management.
+
+### Types of ephemeral volumes
+
+Kubernetes supports several different kinds of ephemeral volumes for
+different purposes:
+- [emptyDir]((/docs/concepts/volumes/#emptydir): empty at Pod startup,
+  with storage provided locally from the root disk or RAM
+- [configMap](/docs/concepts/volumes/#configmap),
+  [downwardAPI](/docs/concepts/volumes/#downwardapi),
+  [secret](/docs/concepts/storage/volumes/#secret): inject different
+  kinds of Kubernetes data into a pod
+- [CSI ephemeral
+  volumes](docs/concepts/storage/volumes/#csi-ephemeral-volumes):
+  similar to the previous volume kinds, but provided by special [CSI
+  drivers](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+  which specifically [support this feature](https://kubernetes-csi.github.io/docs/drivers.html)
+- [generic ephemeral volumes](#generic-ephemeral-volumes), which
+  can be provided by all storage drivers that also support persistent volumes
+
+`emptyDir`, `configMap`, `downwardAPI`, `secret` are provided as
+[local ephemeral
+storage](/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage).
+They are managed by kubelet on each node.
+
+CSI ephemeral volumes *must* be provided by third-party CSI storage
+drivers. Generic ephemeral volumes *can* be provided by third-party
+CSI storage drivers, but also by any other storage driver that
+supports dynamic provisioning. These drivers can offer functionality
```

Comment:
"These drivers" became a bit ambiguous. I replaced it with "The advantage of using third-party drivers is that they can..."